### PR TITLE
Jellyvision 6Sense impact dashboard: account_current_state, buying_stages_movement

### PR DESCRIPTION
**Pull request by izztsfn**
Prepared two separate scripts: db_6sense_account_current_state.sql and db_6sense_buying_stages_movement.sql for gold std script purposes. These will eventually be consolidated into 6sense_impact_dashboard.sql with the remaining chunks of the script

Changes made:
1. Added initial version of the scripts (that match in the scheduler)
2. Removed subqueries
3. Fixed formatting - indentation, spacing
4. Replaced DDL with DML statement 
5. Updated formatting of field names

### CODE REVIEW

File: Clients/Jellyvision/db_6sense_account_current_state.sql [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        CASE
+          WHEN _extractdate LIKE '0%' THEN PARSE_DATE('%m/%d/%y', _extractdate)
+          ELSE PARSE_DATE('%m/%d/%Y', _extractdate)
+        END
+      ) AS _added_on,
+      CONCAT(_6sensecountry, _6sensecompanyname) AS _country_account
+    FROM `x-marketing.jellyvision_mysql.jellyvision_db_segment_target_account`
+    WHERE LENGTH(_extractdate) > 0
+    GROUP BY
+      2
```
Comment (raaayden) [inline Clients/Jellyvision/db_6sense_account_current_state.sql:60]: On line 60 remove the 2 and change it to the actual field and move it to line 59

File: Clients/Jellyvision/db_6sense_account_current_state.sql [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+      --     main._6sensecountry = side._country
+      ) --    OR (
+      --             side._domain NOT LIKE CONCAT('%', SPLIT(main._6sensedomain, '.')[SAFE_OFFSET(0)], '%')
+      --         AND 
+      --             main._6sensecompanyname = side._account_name
+      --     ) 
+    GROUP BY
+      2,
+      3,
+      4
```
Comment (raaayden) [inline Clients/Jellyvision/db_6sense_account_current_state.sql:147]: similar to above, change the group by formatting

**Review by raaayden — COMMENTED** [review]
@izztsfn I have added new comments do check on the buying_stages_movement script as well for the group by

**Comment by izztsfn** [comment]
@raaayden I have updated the formatting for GROUP BY along with PARTITION BY and ORDER BY as well

**Review by raaayden — APPROVED** [review]
Everything looks okay so far !

**Pull request merged**